shelter/README.md

```diff
@@ -0,0 +1,72 @@
+# shelter
+
+Shelter is designed as a remote attestation tool for customer to verify if their workloads are loaded in a specified intel authorized sgx enclaved.
+The verifying process is as below:
+1. shelter setup a security channel based on mTLS with runE enclaved
+2. runE enclaved will generate/retrieve the quote info of workload running enclave
+3. runE enclaved will get IAS report by quote info from Intel authorized web server
+4. runE enclaved will generate attestation verification report
+5. shelter will verify the attestation verification report and mrenclave value by mTLS security channel
+6. shelter will report the verifying result
+
+**Prerequisite**
+	Go 1.13.x or above.
+
+**Build**
+
+Please follow the command to build Inclavare Containers from the latested source code on your system.
+1. Download the latest source code of Inclavare Containers
+
+   mkdir -p "$WORKSPACE"
+
+   cd "$WORKSPACE"
+
+   git clone https://github.com/alibaba/inclavare-containers
```

Comment:
Use '```' to enclose the shell commands.

Reply:
This format without enclose aligns with runE current README, so no need to modify it.

Reply:
No. You are writing markdown. markdown is not the final output format. The README you are seeing is the rendered result from markdown.

Reply:
Change to:

```shell
mkdir -p "$WORKSPACE"
cd "$WORKSPACE"
git clone https://github.com/alibaba/inclavare-containers
```

No need to add extra empty lines.